### DISCUSSION TITLE
What happens with Tenure when there is a change of contract owner (but the end user and MNO may still be the same)

### DISCUSSION THREAD

**Discussion by HuubAppelboom** [camaraproject/Tenure#8] [Q&A]
In case I use  mobile number under an enterprise agreement (my employer  pays for the subscription), and I leave the company and take my number with me, and the contract remains with the MNO, will you reset the Tenure clock ?

In this case the end user remains the same, but the contract owner changes.

The same may occur when one of my children is using a mobile phone number under a contract in my name, and he/she will have to pay teh contract herself, and put the contract in her name, will we reset the tenure clock ?

**Comment by eric-murray** [comment]
Converted to discussion. If an issue is identified, the issue can be re-opened.

**Comment by Noel-Wirzius-DT** [comment]
Thanks Huub for starting this. I want to add some scenarios: 
- Move Over from one Contract type - PrePaid to Postpaid -> same customer, new Tenure Date? 
- Number Takeover from another operator -> This will also call a new Tenure Date, but maybe for some Fraud scenarios it is then not valid, as it is not a new "customer" in general behind this number.

**Reply by HuubAppelboom** [reply]
From a fraud perspective, I don't think the move from prepaid to postpaid will change much , so I don't think you should reset the tenure clock

Number takeover form another operator - sound like a nice idea, but may be too costly to implement on the short term.  If you go on the path of porting this kind of data, this may become complex, and open a box of pandora of other data that needs to be ported as well....

**Reply by KevScarr** [reply]
I would agree with the PrePaid to Postpaid and vice-versa tenure date staying the same (and even for small Business Accounts where you know the customer, also staying the same).

One case to consider. Where you have large Multi-National Accounts  (> 50,000 business numbers where you don't know the 'user' for a MSISDN).  if an employee of that company leaves but takes that Business number with them and converts to Postpaid, which date should be returned. I'm inclined to say the new Postpaid start date for tenure ??
